docs/configuration/module-config/canned-message.mdx

```diff
@@ -1,6 +1,7 @@
 ---
 id: canned-message
 title: Canned Message Module Configuration
+slug: /settings/moduleconfig/canned-message
```

Comment:
On the `moduleconfig` slugs, should we keep it in line with the folder naming convention? `module-config`?

Reply:
> 

I don't really like the idea of breaking all the config links, the current slug matches the protobuf structure and has been linked all over the forum and discord recently.

Reply:
Sounds good, we'll leave it as is. It's just a cosmetic change that has big implications.

If we do change things, I think the `settings/bluetooth` `settings/modules/telemetry` or similar would be ideal.